src/System.IO.Ports/tests/SerialPort/AbortOnError.cs

```diff
@@ -28,7 +29,12 @@ public class AbortOnError : PortsTest
         /// 3. Verify that the fAbortOnError flag is clear
         /// 
         /// </summary>
-        [ConditionalFact(nameof(HasOneSerialPort))]
+        /// <remarks>
+        /// The tests require access, via reflection, to internal type SerialStream and respective methods GetDcbFlag and
+        /// SetDcbFlag, however, that requires either changes to the public type (incresing its size) or to the test itself.
+        /// Tracking issue at https://github.com/dotnet/corefx/issues/23234.
+        /// </remarks>
+        [ConditionalFact(nameof(HasOneSerialPort), nameof(s_isNotUap))]
```

Comment:
the value of activeissue is it makes it possible to check for issues that should not have been closed. as you have written it, there is no good way to detect that 23234 should not have been closed without removing the condition on the test.